### ISSUE TITLE
Automatically ignore language-specific block syntax

### ISSUE THREAD

**Issue by silverwind** [editorconfig-checker/editorconfig-checker#272]
Given this CSS file

```css
/* block
   comment */
.foo {
  color: red;
}
```

and this `.editorconfig`:

```ini
[*]
indent_size = 2
```

It will produce an error `Wrong amount of left-padding spaces(want multiple of 2)` on line 2.

While I understand that editorconfig is language-neutral, I feel like this linter could support block syntax for at least some of the well-known languages. I certainly don't feel like putting exclude comments around every block comment in such cases.

Things that come to mind:

- `/* */` in many languages
- `"""` in python
- ``` ` ``` in javascript and typescript

Prior art includes [eclint](https://github.com/jednano/eclint) which allowed to define [start and end markers](https://github.com/jednano/eclint#support-for-doc-comments) as editorconfig attributes, but I feel like this is something the linter itself should handle.

**Comment by mstruebing** [comment]
I don't think we should implement this.
Editorconfig is (programming) language agnostic and should not know anything about syntax in my opinion.
Otherwise, we open the can of worms and need to provide parser and analyzer for every programming language out there.

I'll close this issue for now.
If you think this is still something that should be implemented feel free to reopen.

**Comment by silverwind** [comment]
Well, without this, it will never be possible to apply editorconfig to all files in a project, only a subset of files that don't have multiline syntaxes. I think it's a mistake trying to be language-agnostic.

**Comment by mstruebing** [comment]
You can indent the comment accordingly or remove this specific line to be checked.

in the future it may even be possible to ignore whole blocks or only the next line.

your editor should intend it correctly if its setup to respects your editorconfig.

When we implement this, we would need to implement an infinite and growing amount of special cases for languages, alone right now I know 6 different commenting styles out of my head and I don’t know any language by far, then there are additional special cases in each language etc. it’s just getting to complex in my opinion.
 
 
If someone can create a PR with an elegant solution I will definitely look at it but I think it shouldn’t be the goal of this tool to implement special cases for specific languages and rather stick to a plain, syntax free language format.
